File: src/api/Yu.Data/Entities/WorkFlow/WorkFlowFormElement.cs
using System;
using System.Collections.Generic;
using System.Text;
using Yu.Data.Infrasturctures;
using Yu.Data.Infrasturctures.BaseIdentity;

namespace Yu.Data.Entities.WorkFlow
{
    [BelongTo(typeof(BaseIdentityDbContext))]
    public class WorkFlowFormElement : BaseEntity<Guid>
    {
        // 工作流定义ID
        public Guid DefineId { get; set; }

        // dom的id
        public string ElementId { get; set; }

        // 元素类型
        public string Type { get; set; }

        // 宽度
        public int Width { get; set; }

        // 行数
        public int Line { get; set; }

        // 选项
        public string Options { get; set; }
    }
}

File: src/client/src/app/workflow/models/workflowFormElement.ts
export class WorkFlowFormElement {
    id?:string;          // id

    name:string;        // 名称

    elementId: string;  // domid

    defineId:string;    // 工作流定义ID
    type: string;       // 类型
    width: number;      // 宽度

    options:string;     // 选项  选择器用

    line: number;       // 行数 文本域用
}